packages/aws-cdk-lib/aws-rds/lib/aurora-cluster-instance.ts

```diff
@@ -484,6 +502,12 @@ class AuroraClusterInstance extends Resource implements IAuroraClusterInstance {
       throw new Error('`enablePerformanceInsights` disabled, but `performanceInsightRetention` or `performanceInsightEncryptionKey` was set');
     }
 
+    this.performanceInsightsEnabled = enablePerformanceInsights;
+    this.performanceInsightRetention = enablePerformanceInsights
+      ? (props.performanceInsightRetention || PerformanceInsightRetention.DEFAULT)
+      : undefined;
+    this.performanceInsightEncryptionKey = props.performanceInsightEncryptionKey;
```

Comment:
Why is this needed instead of the old way

Reply:
This is just to set it to new variables added to the `AuroraClusterInstance` instance variables.

L467-469
```ts
  public readonly performanceInsightsEnabled: boolean;
  public readonly performanceInsightRetention?: PerformanceInsightRetention;
  public readonly performanceInsightEncryptionKey?: kms.IKey;
```